### PR TITLE
FAF/Molecule tags

### PR DESCRIPTION
**Pull request by matt-horwood-mayden**
feat: add username and molecule run ID to name

### CODE REVIEW

**Review by BenMayden — APPROVED** [review]
L's G™

**Review by tdmayden — COMMENTED** [review]
i don't understand this but happy to support the approval if someone does an ELI5!

**Review by tdmayden — APPROVED** [review]
lgtm

**Comment by matt-horwood-mayden** [comment]
> i don't understand this but happy to support the approval if someone does an ELI5!

this will update the Name tag with the name of the `instance name - the name of the user running ansible - the ID of the molecule run`
So that we know who has left random things about
